[Build Issue] addons:jre-libbluray does not build on aarch64

### Describe the bug
Building `addons:jre-libbluray` on Ampere A1 fails.

Wrong jdk (if cross required)

```
[1;36mUNPACK[0m      libbluray
    [1;32mAPPLY PATCH [1;37m(common)[0m[0m      packages/multimedia/libbluray/patches/libbluray-00-fix-build-with-autoconf-archive-2016-or-later.patch
patching file Makefile.am
    [1;32mAPPLY PATCH [1;37m(common)[0m[0m      packages/multimedia/libbluray/patches/libbluray-03-set-headless-false.patch
patching file src/libbluray/bdj/bdj.c
Hunk #1 succeeded at 920 (offset 17 lines).
    [1;33mFIXCONFIG[0m      /build/build.LibreELEC-ARMv7.arm-12.0-devel/build/libbluray-1.3.4/build-aux/
[1;36mUNPACK[0m      jre-libbluray
    [1;32mAPPLY PATCH [1;37m(absolute - /build/packages/multimedia/libbluray/patches/libbluray-00-fix-build-with-autoconf-archive-2016-or-later.patch)[0m[0m      packages/multimedia/libbluray/patches/libbluray-00-fix-build-with-autoconf-archive-2016-or-later.patch
patching file Makefile.am
    [1;32mAPPLY PATCH [1;37m(absolute - /build/packages/multimedia/libbluray/patches/libbluray-03-set-headless-false.patch)[0m[0m      packages/multimedia/libbluray/patches/libbluray-03-set-headless-false.patch
patching file src/libbluray/bdj/bdj.c
Hunk #1 succeeded at 920 (offset 17 lines).
    [1;33mFIXCONFIG[0m      /build/build.LibreELEC-ARMv7.arm-12.0-devel/build/jre-libbluray-1.3.4/build-aux/
[1;33mBUILD[0m      jre-libbluray [1;37m(target)[0m
    [1;35mTOOLCHAIN[0m      autotools
    [1;35mAUTORECONF[0m      jre-libbluray
autoreconf: export WARNINGS=
autoreconf: Entering directory '/build/build.LibreELEC-ARMv7.arm-12.0-devel/build/jre-libbluray-1.3.4'
autoreconf: configure.ac: not using Gettext
autoreconf: running: /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/aclocal -I /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/armv7a-libreelec-linux-gnueabihf/sysroot/usr/share/aclocal -I /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/armv7a-libreelec-linux-gnueabihf/sysroot/usr/share/aclocal --force -I m4
autoreconf: configure.ac: tracing
autoreconf: running: /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/libtoolize --copy --force
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, 'build-aux'.
libtoolize: copying file 'build-aux/ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
autoreconf: configure.ac: not using Intltool
autoreconf: configure.ac: not using Gtkdoc
autoreconf: running: /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/aclocal -I /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/armv7a-libreelec-linux-gnueabihf/sysroot/usr/share/aclocal -I /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/armv7a-libreelec-linux-gnueabihf/sysroot/usr/share/aclocal --force -I m4
autoreconf: running: /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/autoconf --include=/build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/armv7a-libreelec-linux-gnueabihf/sysroot/usr/share/aclocal --force
autoreconf: running: /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/autoheader --include=/build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/armv7a-libreelec-linux-gnueabihf/sysroot/usr/share/aclocal --force
autoreconf: running: /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/automake --add-missing --copy --force-missing
configure.ac:74: installing 'build-aux/compile'
configure.ac:24: installing 'build-aux/missing'
Makefile.am: installing 'build-aux/depcomp'
autoreconf: Leaving directory '/build/build.LibreELEC-ARMv7.arm-12.0-devel/build/jre-libbluray-1.3.4'
Executing (target): /build/build.LibreELEC-ARMv7.arm-12.0-devel/build/jre-libbluray-1.3.4/configure --host=armv7a-libreelec-linux-gnueabihf --build=aarch64-linux-gnu --prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --libdir=/usr/lib --libexecdir=/usr/lib --localstatedir=/var --disable-static --enable-shared --disable-werror --disable-extra-warnings --disable-optimizations --disable-examples --enable-bdjava-jar --disable-doxygen-doc --disable-doxygen-dot --disable-doxygen-man --disable-doxygen-rtf --disable-doxygen-xml --disable-doxygen-chm --disable-doxygen-chi --disable-doxygen-html --disable-doxygen-ps --disable-doxygen-pdf --with-freetype --with-fontconfig --with-libxml2 --with-gnu-ld
checking build system type... aarch64-unknown-linux-gnu
checking host system type... armv7a-libreelec-linux-gnueabihf
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for armv7a-libreelec-linux-gnueabihf-strip... /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-strip
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether UID '1000' is supported by ustar format... yes
checking whether GID '1000' is supported by ustar format... yes
checking how to create a ustar tar archive... gnutar
checking whether make supports nested variables... (cached) yes
checking whether make supports the include directive... yes (GNU style)
checking for armv7a-libreelec-linux-gnueabihf-gcc... /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc accepts -g... yes
checking for /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc option to enable C11 features... none needed
checking whether /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc understands -c and -o together... yes
checking dependency style of /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc... gcc3
checking for getfsstat... no
checking for getvfsstat... no
checking for armv7a-libreelec-linux-gnueabihf-gcc... (cached) /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc
checking whether the compiler supports GNU C... (cached) yes
checking whether /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc accepts -g... (cached) yes
checking for /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc option to enable C11 features... (cached) none needed
checking whether /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc understands -c and -o together... (cached) yes
checking dependency style of /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc... (cached) gcc3
checking for armv7a-libreelec-linux-gnueabihf-ar... /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-ar
checking the archiver (/build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-ar) interface... ar
checking how to print strings... printf
checking for a sed that does not truncate output... /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/sed
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for fgrep... /usr/bin/grep -F
checking for ld used by /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc... /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-ld
checking if the linker (/build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-nm
checking the name lister (/build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-nm) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking how to convert aarch64-unknown-linux-gnu file names to armv7a-libreelec-linux-gnueabihf format... func_convert_file_noop
checking how to convert aarch64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-ld option to reload object files... -r
checking for armv7a-libreelec-linux-gnueabihf-file... no
checking for file... file
configure: WARNING: using cross tools not prefixed with host triplet
checking for armv7a-libreelec-linux-gnueabihf-objdump... /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-objdump
checking how to recognize dependent libraries... pass_all
checking for armv7a-libreelec-linux-gnueabihf-dlltool... no
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for armv7a-libreelec-linux-gnueabihf-ar... (cached) /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-ar
checking for archiver @FILE support... @
checking for armv7a-libreelec-linux-gnueabihf-strip... (cached) /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-strip
checking for armv7a-libreelec-linux-gnueabihf-ranlib... /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-ranlib
checking command to parse /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-nm output from /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc object... ok
checking for sysroot... no
checking for a working dd... /usr/bin/dd
checking how to truncate binary pipes... /usr/bin/dd bs=4096 count=1
checking for armv7a-libreelec-linux-gnueabihf-mt... no
checking for mt... no
checking if : is a manifest tool... no
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -fno-rtti -fno-exceptions... no
checking for /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc option to produce PIC... -fPIC -DPIC
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc PIC flag -fPIC -DPIC works... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc static flag -static works... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -c -o file.o... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -c -o file.o... (cached) yes
checking whether the /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc linker (/build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-ld) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking for stdarg.h... yes
checking for sys/types.h... (cached) yes
checking for dirent.h... yes
checking for errno.h... yes
checking for libgen.h... yes
checking for malloc.h... yes
checking for stdlib.h... (cached) yes
checking for mntent.h... yes
checking for inttypes.h... (cached) yes
checking for strings.h... (cached) yes
checking for sys/time.h... yes
checking for time.h... yes
checking for mntent.h... (cached) yes
checking for dirent.h that defines DIR... yes
checking for library containing opendir... none required
checking for struct dirent.d_type... yes
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... 64
checking for snprintf... yes
checking for pthread.h... yes
checking for library containing pthread_create... none required
checking for library containing dlopen... none required
checking for dladdr... yes
checking pkg-config is at least version 0.9.0... yes
checking for libxml-2.0 >= 2.6... yes
checking for freetype2... yes
checking for fontconfig... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -Wall flag... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -Wdisabled-optimization flag... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -Wpointer-arith flag... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -Wredundant-decls flag... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -Wcast-qual flag... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -Wwrite-strings flag... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -Wtype-limits flag... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -Wundef flag... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -Wmissing-prototypes flag... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -Wshadow flag... yes
checking if /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc supports -Werror=implicit-function-declaration flag... yes
checking for /build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-gcc way to treat warnings as errors... -Werror
checking for ant... yes
checking for javac... /build/build.LibreELEC-ARMv7.arm-12.0-devel/build/jdk-x86_64-zulu-8.38.0.13-8.0.212/bin/javac
checking for libudfread >= 1.1.0... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating src/libbluray.pc
config.status: creating src/libbluray/bluray-version.h
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands
  Summary:
  --------
  BD-J type:                     j2se
  build JAR:                     yes
      javac version:             /build/build.LibreELEC-ARMv7.arm-12.0-devel/build/jre-libbluray-1.3.4/configure: line 15867: /build/build.LibreELEC-ARMv7.arm-12.0-devel/build/jdk-x86_64-zulu-8.38.0.13-8.0.212/bin/javac: cannot execute binary file: Exec format error
      Java binary version:       1.4
  Font support (freetype2):      yes
  Use system fonts (fontconfig): yes
  Metadata support (libxml2):    yes
  External libudfread:           yes
  Build examples:                no
ant -f /build/build.LibreELEC-ARMv7.arm-12.0-devel/build/jre-libbluray-1.3.4/src/libbluray/bdj/build.xml \
    -Dbuild='/build/build.LibreELEC-ARMv7.arm-12.0-devel/build/jre-libbluray-1.3.4/.armv7a-libreelec-linux-gnueabihf/src/libbluray/bdj/build' \
    -Ddist='/build/build.LibreELEC-ARMv7.arm-12.0-devel/build/jre-libbluray-1.3.4/.armv7a-libreelec-linux-gnueabihf/.libs' \
    -Dsrc_awt=:java-j2se:java-build-support \
    -Djava_version_asm=1.5 -Djava_version_bdj=1.4 \
    -Dbootclasspath="" \
    -Dversion='j2se-1.3.4'
/build/build.LibreELEC-ARMv7.arm-12.0-devel/toolchain/bin/ant: 1: exec: /build/build.LibreELEC-ARMv7.arm-12.0-devel/build/jdk-x86_64-zulu-8.38.0.13-8.0.212/jre/bin/java: Exec format error
make: *** [Makefile:2800: all-local] Error 126
[1;31mFAILURE: scripts/build jre-libbluray:target during make_target (package.mk)[0m
*********** FAILED COMMAND ***********
make all-local
**************************************
[1;31mFAILURE: scripts/build jre-libbluray:target has failed![0m
```

--- a/packages/addons/addon-depends/jre-depends/apache-ant/package.mk
+++ b/packages/addons/addon-depends/jre-depends/apache-ant/package.mk
@@ -8,7 +8,7 @@ PKG_SHA256="de0a860bf339e6975558f426772a6aa2c6ae0856215dd9aaf57aa2476d665664"
 PKG_LICENSE="Apache License 2.0"
 PKG_SITE="https://ant.apache.org/"
 PKG_URL="https://archive.apache.org/dist/ant/binaries/${PKG_NAME}-${PKG_VERSION}-bin.tar.xz"
-PKG_DEPENDS_UNPACK="jdk-x86_64-zulu"
+PKG_DEPENDS_UNPACK="jdk-${MACHINE_HARDWARE_NAME}-zulu"
 PKG_LONGDESC="Apache Ant is a Java library and command-line tool that help building software."
 PKG_TOOLCHAIN="manual"
 

--- a/packages/addons/addon-depends/jre-depends/jre-libbluray/package.mk
+++ b/packages/addons/addon-depends/jre-depends/jre-libbluray/package.mk
@@ -7,7 +7,7 @@ PKG_NAME="jre-libbluray"
 PKG_DEPENDS_TARGET+=" apache-ant:host"
 PKG_LONGDESC="libbluray jar for BD-J menus"
 PKG_URL=""
-PKG_DEPENDS_UNPACK+=" jdk-x86_64-zulu libbluray"
+PKG_DEPENDS_UNPACK+=" jdk-${MACHINE_HARDWARE_NAME}-zulu libbluray"
 PKG_PATCH_DIRS+=" $(get_pkg_directory libbluray)/patches"
 PKG_BUILD_FLAGS="-sysroot"
 
@@ -17,7 +17,7 @@ unpack() {
 }
 
 pre_configure_target() {
-  export JAVA_HOME="$(get_build_dir jdk-x86_64-zulu)"
+  export JAVA_HOME="$(get_build_dir jdk-${MACHINE_HARDWARE_NAME}-zulu)"
 
   # build also jar
   PKG_CONFIGURE_OPTS_TARGET="${PKG_CONFIGURE_OPTS_TARGET/disable-bdjava-jar/enable-bdjava-jar}"

